docs/streams/core-concepts.html

```diff
@@ -172,6 +172,37 @@ <h2><a id="streams_processing_guarantee" href="#streams_processing_guarantee">Pr
         More details can be found in the <a href="/{{version}}/documentation#streamsconfigs"><b>Kafka Streams Configs</b></a> section.
     </p>
 
+    <h3><a id="streams_out_of_ordering" href="#streams_out_of_ordering">Out-of-Order Handling</a></h3>
+
+    <p>
+        Besides the guarantee that each record will be processed exactly-once, another issue that many stream processing application will face is how to
+        handle <a href="tbd">out-of-order data</a> data that may impact their business logic. In Kafka Streams, there are two causes that could potentially
```

Comment:
`out-of-order data</a> data` -> duplicate `data`

Reply:
ack.

Reply:
I'm leaving the `tbd` link is for the BIRTE paper reference, once it is publically accessible.